android/app/src/main/AndroidManifest.xml

```diff
@@ -17,7 +18,8 @@
     </queries>
 
    <application
-        android:label="TMail"
+        android:label="@string/app_name"
+        android:requestLegacyExternalStorage="true"
```

Comment:
we also no need this flag @huynguyennovem ?

Reply:
No, you can remove it